bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -105,49 +120,54 @@ pub mod pallet {
 		/// Convert versioned location failure
 		UnsupportedLocationVersion,
 		/// Check location failure, should start from the dispatch origin as owner
-		OwnerCheck,
+		InvalidAssetOwner,
 		/// Send xcm message failure
 		Send,
```

Comment:
Mirror inbound-queue-v2 error types

```suggestion
		SendFailure,
```